index.html

```diff
@@ -1,10 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
-<head> 
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- google fonts link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="index.css" />
+
     <title>Progress Tracker</title>
-    <link rel="stylesheet" href="/index.css">
-</head>
-<body>
-    <h1>Scrimba Bootcamp Progress Tracker</h1>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <!-- header goes here -->
+    <div class="hero_section">
```

Comment:
I would suggest using a `<section>` tag to wrap this entire block of code rather than a generic `<div>`. This helps with readability as well as accessibility and SEO.